### DISCUSSION TITLE
How do I make a png font for this?

### DISCUSSION THREAD

**Discussion by YangAnimationsTWO** [Techokami/SonicWorldsNext#208] [General]
I need to do the S1 Card fonts, but idk how to make the png a font
![S1CardFont](https://github.com/user-attachments/assets/9f528c2d-0e81-4aa9-9628-3e84955718da)

**Comment by Renhoex** [comment]
this section on bitmap fonts should help
https://docs.godotengine.org/en/stable/tutorials/ui/gui_using_fonts.html
